distribution/src/main/release/samples/pom.xml

```diff
@@ -99,6 +99,7 @@
         <module>wsdl_first_rpclit</module>
         <module>jms_pubsub</module>
         <module>jax_rs/spring_security</module>
+        <module>jax_rs/spring_boot</module>
```

Comment:
I only added this one for now.  If you're in agreement, I'll also apply the changes to the auto configure example.